content/en/docs/tasks/inject-data-application/podpreset.md

```diff
@@ -1,22 +1,19 @@
 ---
 reviewers:
 - jessfraz
-title: Inject Information into Pods Using a PodPreset
+title: Inject Information into Pods Using a Pod Preset
 content_template: templates/task
 weight: 60
 ---
```

Comment:
Since you're tidying this page, would you be willing to add `min-kubernetes-server-version` to the front matter?
See https://github.com/kubernetes/website/pull/19265/ for an example of someone doing this.

Reply:
I added the server-version